decidim-admin/app/models/decidim/admin/abilities/base.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    module Abilities
+      # Defines the abilities for a user in the admin section. Intended to be
+      # used with `cancancan`.
+      class Base
```

Comment:
I believe this should be `ParticipatoryProcessAdmin` instead. We should also have an `Admin` ability.

The `Base` ability should only merge those two together.